app/js/lib/foxbox/recipes.js

```diff
@@ -28,8 +28,9 @@ export default class Recipe {
       `${this[p.net].origin}/api/v${this[p.settings].apiVersion}/services`)
       // Get all recipes.
       .then(services => services.filter(service =>
-        service.adapter === 'thinkerbell-adapter' &&
-        service.id !== 'thinkerbell-root-service'
+        service.adapter != 'thinkerbell-adapter'
+        && service.adapter != 'webpush@link.mozilla.org'
```

Comment:
I usually ban `==` and always prefer `===` (or `!==`). I didn't notice that ones. Let's enforce that with eslint.

I placed the operator at the beginning of the new line to avoid polluting the git diff when we add new conditions (only one line change), but I find it weird. Tell me what you prefer and I'll use it.